shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -743,7 +747,24 @@ void TizenRendererEvasGL::SetGeometry(int32_t x,
                                       int32_t y,
                                       int32_t width,
                                       int32_t height) {
-  FT_UNIMPLEMENTED();
+  evas_object_move(evas_window_, x, y);
+  evas_object_resize(evas_window_, width, height);
+
+  evas_object_resize(graphics_adapter_, width, height);
+
+  evas_object_image_native_surface_set(graphics_adapter_, nullptr);
+
+  evas_gl_surface_destroy(evas_gl_, gl_surface_);
+  evas_gl_surface_destroy(evas_gl_, gl_resource_surface_);
+
+  evas_object_image_size_set(graphics_adapter_, width, height);
+  gl_surface_ = evas_gl_surface_create(evas_gl_, gl_config_, width, height);
+  gl_resource_surface_ = evas_gl_pbuffer_surface_create(evas_gl_, gl_config_,
+                                                        width, height, nullptr);
+
+  Evas_Native_Surface ns;
```

Comment:
```suggestion
  Evas_Native_Surface native_surface;
```
https://github.com/flutter-tizen/engine/pull/215/files#diff-7da27c26718646d37e1f0d67f42c107f15b734905da41c2211ae0e0ebcad8828R631-R633